opentelemetry-semantic-conventions/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "opentelemetry-semantic-conventions"
-version = "0.15.0"
+version = "0.16.0"
```

Comment:
version bump is done once before crate publishing, we can revert this change for now.